software/psono.yml

```
@@ -1,11 +1,11 @@
 name: Psono
 website_url: https://psono.com/
-description: A promising password managers fully featured for teams.
+description: Promising password managers fully featured for teams.
```

```suggestion
description: Password manager for companies.
```

Official description from the website